test/conftest.py

```diff
@@ -70,12 +70,10 @@ def args(request):
 
 
 @pytest.fixture(params=[
-    'debian',
     'debian_9',
     'debian_10',
-    'centos7',
```

Comment:
The reason I originally created the `centos7` specific test was due to `basic-install.sh` code specific to `7` as well as specific tests in the test suite for that code (see def suffix `centos_eq_7`). 
My plan was to delete the related code and tests when CentOS 7 is EOL... in 2024 🛸 

While I do like that the test suite is now running on more distro versions, I did not want my origional intent to be lost as to why `centos7` was added into the original `["debian", "fedora", "centos"]`

Reply:
I guess you marked this as resolved as you saw the `centos_7` rename? 😏

Reply:
I marked it resolved because this is more of an fyi to maintainers and no changes required. Just wanted to mention this because there was actually a code-base related reason that I was testing centos7 separately and not just because we _support_ it.